src/ngx_http_small_light_imagemagick.c

```diff
@@ -183,6 +155,34 @@ ngx_int_t ngx_http_small_light_imagemagick_process(ngx_http_request_t *r, ngx_ht
         DestroyPixelWand(bg_color);
     }
 
+    /* calc size. */
+    iw = (double)MagickGetImageWidth(ictx->wand);
+    ih = (double)MagickGetImageHeight(ictx->wand);
+    ngx_http_small_light_calc_image_size(r, ctx, &sz, iw, ih);
+
+    /* pass through. */
```

Comment:
Would you move this processing to the underneath line#95?

https://github.com/gunyoki/ngx_small_light/blob/fix_rotation/src/ngx_http_small_light_imagemagick.c#L95